packages/lib-classifier/src/plugins/drawingTools/models/tools/Tool/Tool.js

```diff
@@ -90,6 +90,10 @@ const Tool = types
       mark.initialDrag?.(event)
     },
 
+    // this outputs the mouse coords when not creating (ig: guideline for Polygon)
+    // By default, this does nothing. Must be implemented in model/tool file (polygonTool.js)
+    handlePointerPosition(event, mark) {},
+
```

Comment:
[documentation] I strongly recommend that you also update https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/drawingTools/README.md with the new `handlePointerPosition` info.

But 👍 for having inline documentation too!

Reply:
Added ✅